lib/generators/templates/devise.rb

```diff
@@ -297,7 +297,8 @@
   # config.omniauth_path_prefix = '/my_engine/users/auth'
 
   # ==> Turbolinks configuration
-  # If your app is using Turbolinks, Turbolinks::Controller needs to be included to make redirection work correctly:
+  # If your app is using Turbolinks (but *not* Turbo), Turbolinks::Controller needs
```

Comment:
Do we need to tell that Turbolinks is different of Turbo? I thought it was clear by the name that requirements for one are different of the other.

Reply:
Maybe it's just me, but I assumed that the requirements would be the same since Turbo (Drive) is being pitched an evolution of Turbolinks.

Reply:
If I'm not mistaken this doesn't even work for current (now deprecated) Turbolinks, but only for Turbolinks Classic.